packages/cli/src/util/projects/get-project-by-cwd-or-link.ts

```diff
@@ -26,27 +25,10 @@ export default async function getProjectByCwdOrLink({
     return project;
   }
 
-  const pathValidation = await validatePaths(client, [cwd]);
-  if (!pathValidation.valid) {
-    if (pathValidation.exitCode) {
-      const err: NodeJS.ErrnoException = new Error(
-        'Invalid current working directory'
-      );
-      err.code = 'ERR_INVALID_CWD';
-      throw err;
-    }
-    const err: NodeJS.ErrnoException = new Error('Canceled');
-    err.code = 'ERR_CANCELED';
-    throw err;
-  }
-
   // ensure the current directory is a linked project
-  const linkedProject = await ensureLink(
-    commandName,
-    client,
-    pathValidation.path,
-    { autoConfirm }
-  );
+  const linkedProject = await ensureLink(commandName, client, cwd, {
```

Comment:
**nitpick (non-blocking):** ‏I think the old formatting with params on separate lines is easier to read that the changed code.

Reply:
Prettier <picture data-single-emoji=":blob_shrug:" title=":blob_shrug:"><img class="emoji" src="https://emoji.slack-edge.com/T0CAQ00TU/blob_shrug/5994f7af73108186.png" alt=":blob_shrug:" width="20" height="auto" align="absmiddle"></picture>